core/src/edu/asu/c3simulator/screens/BusinessDirectonScreen.java

```diff
@@ -0,0 +1,159 @@
+		
+		Label longTermLabel = new Label("Long Term", skin);
+		Label shortTermLabel = new Label("Short Term", skin);
+		
+		optionCheckBoxesGroup = new ButtonGroup(longTermOption, shortTermOption);
+		optionCheckBoxesGroup.setMaxCheckCount(1);
+		optionCheckBoxesGroup.setMinCheckCount(0);
+		optionCheckBoxesGroup.setUncheckLast(true);
+		
+		directions.add(longTermLabel).height(100).top();
+		directions.add(shortTermLabel).height(100).top().spaceLeft(75);
+		directions.row();
+		directions.add(longTermDirection).top();
+		directions.add(shortTermDirection).top().spaceLeft(75);
+		directions.row();
+		directions.add(longTermOption).height(100);
+		directions.add(shortTermOption).height(100).spaceLeft(75);
+		
+		directions.padLeft(100);
+		directions.setTransform(true);
+		
+		addActor(directions, new Location(stage.getWidth() / 2, stage.getHeight() / 2));
+	}
+	
+	/**
+	 * This method creates the long term labels and button that will be displayed in the
+	 * screen.
+	 * 
+	 * @return the GUI that was created.
+	 */
+	private Actor createLongTermDirection()
+	{
+		String communityOutreach[] = { "Community Outreach:",
+				"Building company's name recognition ",
+				"within the community through outreach ", "projects." };
+		String revenue[] = { "Revenue:", "Concrete, annual fiscal goals supporting ",
+				"revenue expectations." };
+		String customerService[] = { "Customer Service:",
+				"Maintain a long-lasting bond and ", "build a devoted following." };
+		
+		VerticalGroup longTermDirectionText = new VerticalGroup();
+		longTermDirectionText.addActor(createParagraph(communityOutreach));
+		longTermDirectionText.addActor(createParagraph(revenue));
+		longTermDirectionText.addActor(createParagraph(customerService));
+		
+		longTermDirectionText.space(25);
+		longTermDirectionText.left();
+		return longTermDirectionText;
+	}
+	
+	/**
+	 * This method creates the short term labels and button that will be displayed in the
+	 * screen.
+	 * 
+	 * @return the GUI that was created.
+	 */
+	private Actor createShortTermDirection()
+	{
+		String communityOutreach[] = { "Community Outreach:",
+				"Incentives for employees who volunteer",
+				"with designated community projects." };
+		String revenue[] = { "Revenue:", "One month contracts for advising",
+				"consultants" };
+		String customerService[] = { "Customer Service:",
+				"Host monthly drawings for free", "products or discounts on future ",
+				"purchases." };
+		
+		VerticalGroup longTermDirectionText = new VerticalGroup();
+		longTermDirectionText.addActor(createParagraph(communityOutreach));
+		longTermDirectionText.addActor(createParagraph(revenue));
+		longTermDirectionText.addActor(createParagraph(customerService));
+		
+		longTermDirectionText.space(25);
+		longTermDirectionText.left();
+		return longTermDirectionText;
+	}
+	
+	/**
+	 * A method that creates the paragraph that will be displayed and organize it in a
+	 * table.
+	 * 
```

Comment:
Get rid of "A method that" and start with "Creates"
It's implied that its a method